test/unit/controller/edit-service-name/post.controller.test.js

```diff
@@ -88,11 +88,11 @@ describe('Controller: editServiceName, Method: get', () => {
       res = {
         redirect: sinon.spy()
       }
-      const result = editServiceNameCtrl.post(req, res)
-      if (result) {
-        done(new Error('Returned a promise'))
-      } else {
-        done()
+
+      try {
+        editServiceNameCtrl.post(req, res)
+      } catch (err) {
+        throw new Error('Returned a promise')
```

Comment:
I don't think this does the same thing as before. We should expect it to return a promise now as it's an async function, so just remove the try/catch